eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -210,22 +210,25 @@ public final Attr attr(final int pos) {
                 )
             );
         }
-        int idx;
-        for (idx = 0; idx < pos; ++idx) {
+        Attr attr = this.attr(this.order.get(0));
+        for (int idx = 0; idx <= pos; ++idx) {
```

Comment:
@mximp Why do you need equality here? Do you have any proofs (tests)?

Reply:
@volodya-lombrozo `idx` iterates through all arg positions up until `pos` inclusive. Previously the iteration was done through all but the `pos` itself and `pos` was queried outside the loop. I've changed it to avoid duplication.

Reply:
@volodya-lombrozo new functional changes are introduced here. This code is covered by existing tests.